quick-start/src/main/java/com/marklogic/hub/web/form/LoginForm.java

```diff
@@ -13,6 +13,8 @@
 	private String mlTracePort;
 	private String mlUsername;
 	private String mlPassword;
+	private String mlAdminUsername;
+    private String mlAdminPassword;
```

Comment:
The whitespace seems to be off in a bunch of places. This ^ is one of them. Please take a look at whitespace to make sure your .editorconfig is working and that you are using 4 spaces in Java files.